### DISCUSSION TITLE
Remote vs Local Assets

### DISCUSSION THREAD

**Discussion by mikhailswift** [zarf-dev/zarf#40] [Q&A] [answered]
Hi all!

I was wondering if I could get some clarification on the difference between local and remote assets.  I think local charts/images/manifests/files seem fairly straight forward: assets that will be deployed onto the air gapped cluster.

Remote images & repos I'm a bit more fuzzy on.  My guess is external assets that local assets may reach out to? Or perhaps assets for use in operating & maintaining the air gapped cluster?

**Comment by RothAndrew** [comment, marked as answer]
`local:` Uses a native k3s capability that enables Kubernetes to use local image tarballs when pulling images instead of needing an image registry.

`remote:` loads the desired images into the private docker registry and repos into the private Gitea that Zarf creates when you initialize with the "Utility Cluster" feature turned on.

See the brand new [Big Bang Core Example](https://github.com/defenseunicorns/zarf/pull/29) for an example of how `remote:` is being used to facilitate GitOps

**Comment by jeff-mccoy** [comment]
Yeah tbh, I actually kind of hate the local/remote designation now and want to talk about removing it as most things are either local or remote only--or it doesn't matter.  The only thing that could be both that I can think of is images.  Also want to evaluate the use of the features to group capabilities for regular zarf packages and not just local packages like `zarf-init`.
